packs/feats/animal-shape.json

```
@@ -24,7 +24,129 @@
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Dinosaur Form (Ankylosaurus)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Spider)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Scorpion)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Mantis)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Centipede)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Beetle)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Insect Form (Ant)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Aerial Form (Wasp)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Aerial Form (Pterosaur)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Aerial Form (Bird)"
+                    },
+                    {
+                        "value": "Compendium.pf2e.spell-effects.Item.Spell Effect: Aerial Form (Bat)"
+                    }
+                ],
+                "flag": "beastkin-animal",
+                "key": "ChoiceSet"
+            },
+            {
+                "actorFlag": true,
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "Yes",
+                        "value": 1
+                    },
+                    {
+                        "label": "No",
+                        "value": 0
+                    }
+                ],
+                "flag": "beastkin-flight",
+                "key": "ChoiceSet",
+                "prompt": "I18N-ME Does your inherent animal have flight speed."
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.beastkin.forms",
+                "predicate": [
+                    "feat:critter-shape"
+                ],
+                "value": {
+                    "label": "Critter",
+                    "value": "critter"
+                }
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.beastkin.forms",
+                "value": {
+                    "label": "Animal",
```

We can use the trait for this
```suggestion
                    "label": "PF2E.TraitAnimal",
```